README.md

```diff
@@ -8,10 +8,20 @@ This is not an official Google product.
 
 Indexing hosts:
  - Linux: supported - follow below documentation.
- - Windows, MacOS: didn't try - backend part likely compiles, wiring and Kythe frontend likely not (see #38).
+ - Windows, MacOS: didn't try - backend part likely compiles, wiring and Kythe
+   frontend likely not (see #38).
 
 Compilers:
-  - GHC 7.10.3, 8.0.1, 8.0.2, 8.2.1
+  - GHC 8.0.2
+  - GHC 8.2.2
+  - GHC 8.4.3
+
+Stackage:
+  - A recent LTS release corresponding to above compilers is supported.
+    See `stack-ghcXXX.yml` files.
+
+Previous compilers (from GHC 7.10.3) were supported at some point. Look at
```

Comment:
Can we add git tags at the versions which drop a GHC version?
(if people want the older version, they simply checkout that git tag and have everything working)

Reply:
Good idea. Created https://github.com/google/haskell-indexer/releases/tag/last-ghc-7.10.3 .